devito/ops/transformer.py

```diff
@@ -201,8 +207,24 @@ def create_ops_dat(f, name_to_ops_dat, block):
     return res
 
 
-def create_ops_par_loop(
-        trees, ops_kernel, parameters, block, name_to_ops_dat, par_to_ops_stencil, dims):
+def create_ops_fetch(f, name_to_ops_dat, time_upper_bound):
+
+    if f.is_TimeFunction:
+        ops_fetch = [namespace['ops_dat_fetch_data'](
+            name_to_ops_dat[f.name].indexify(
+                [Mod(Add(time_upper_bound, -i), f._time_size)]),
+            Byref(f.indexify([Mod(Add(time_upper_bound, -i), f._time_size)])))
+            for i in range(f._time_size)]
+
+    else:
+        ops_fetch = [namespace['ops_dat_fetch_data'](
+            name_to_ops_dat[f.name], Byref(f.indexify([0])))]
```

Comment:
0?

Reply:
As explained, this is an hack. 

I needed to access a pointer to the beginning of an array. But I couldn't generate just `v`, instead, I managed to generate `&(v[0])`.
As you have suggested, I added a comment here to explain this.